NewForce-Cohort-4/tabloidmvc-team-neurotic-ninjas
# Add and Remove Tags from a Post

# Description
This PR includes Tickets 17 and 18

Ticket 17
As an author I would like to be able to associate one or more Tags with one of my Posts so that readers can easily see the Post topics at a glance.
 
**Given** the author is viewing Post details
**When** they select the `Manage Tags` option
**Then** they should be presented with a selection of Tags options to associate with the Post
 
**Given** the author has selected the Tags they wish to associate with a Post
**When** they click the `Save` button
**Then** the association should be saved to the database
**And** the user should be directed to the Post details page
**And** the selected Tags should be displayed on the Post details page

Ticket 18
As an author I would like to be able to unassociate one or more Tags from one of my Posts so that I can correct a mistakenly added Tag.
 
**Given** the author is viewing Post details
**When** they select the `Manage Tags` option
**Then** they should be presented with a selection of Tags options that are associated with the Post
 
**Given** the author has selected the Tags they wish to remove from a Post
**When** they click the `Save` button
**Then** the association should be removed from the database
**And** the user should be directed to the Post details page
**And** the removed Tags should no longer be displayed on the Post details page

## Type of change
Please delete options that are not relevant.

# Testing Instructions
- pull down the dtb-remove-tags branch and start the application
- Login and navigate to the Posts tab
- Click on the eye button view the details of a post
- Once in the Post Details view, select the Manage Tags button
- In the left menu, above the green Save button, select one or more tags to add to the post. Click Save.
- You should be navigated back to the Post Details view, where the tags that you saved should be present along the top of the page.
- Select the Manage Tags button once again.
- This time, use the right menu to remove one or more tags from your post. Click Remove.
- You should be navigated back to the Details view where you should see that the tags were removed.

# Checklist:
